content/maxwell3_fdem/sphere_freespace/dipole_response.rst

```diff
@@ -0,0 +1,91 @@
+.. _dipole_response
+
+The Sphere's Dipolar Response
+-----------------------------
+
+**Purpose**: Here, expressions from Wait (1951) presented for predicting the dipole response from a conductive and magnetically permeable sphere in free-space.
+We show how the characteristic response of the sphere is dependent on its excitation factor; where the excitation factor is used to define the induced dipole moment of the sphere.
+
+.. figure:: ./images/figFreeSpaceSphere.png
+        :align: right
+        :figwidth: 40%
+        :name: FreeSpaceSphere
+
+        Problem geometry for a conductive and magnetically permeable sphere in free-space.
+
+Consider the problem geometry illustrated in :numref:`FreeSpaceSphere`, wherein a sphere of radius :math:`R`, conductivity :math:`\sigma` and magnetic permeability :math:`\mu` is located in the vicinity of an inductive source transmitter (Tx).
```

Comment:
for ease of picking out variables, I would find it helpful to have them in a bullet point list instead of a paragraph. ie. 
```
- :math:`R` : radius
- :math:`\sigma` : electrical conductivity
- ... 
```